### ISSUE TITLE
Not showing up in sidebar

### ISSUE THREAD

**Issue by g7nbp** [dzaman/google-meet-transcripts#2]
The extension is loaded and has access to the page, but is not showing up in the sidebar as expected - I see only participants and chat, not the transcribe button
![Screenshot 2020-06-19 at 15 38 17](https://user-images.githubusercontent.com/38110470/85144506-2619cf00-b243-11ea-802e-5893a2fe8d29.png)

Console log shows :

[google-meet-transcripts] loaded
stenographer.js:1168 
[google-meet-transcripts] localStorage version 1

Doesn't look like the addButton funct is running.

I am using Version 83.0.4103.112 (Official Build) on Chromebook.

**Comment by g7nbp** [comment]
![Screenshot 2020-06-19 at 16 24 11](https://user-images.githubusercontent.com/38110470/85150206-27022f00-b24a-11ea-8b2b-040e27c43654.png)

Nothing showing up in the div  from the addButtonLoop

**Comment by g7nbp** [comment]
Looks like the funct to find the div in the DOM is not finding it.

**Comment by Nu11u5** [comment]
Meet changed the labels for the existing toolbar buttons.  Changing the related code and sideloading the extension in developer mode shows that this fixes the issue.

``` diff
diff --git a/src/stenographer.js b/src/stenographer.js
index d9b7665..0158c1b 100644
--- a/src/stenographer.js
+++ b/src/stenographer.js
@@ -946,8 +946,8 @@ try {
   // Find the button container in the DOM
   // -------------------------------------------------------------------------
   const findButtonContainer = () => {
-      const participantsIconXpath = `//div[@aria-label='Show participant options']//*[@d='M15 8c0-1.42-.5-2.73-1.33-3.76.42-.14.86-.24 1.33-.24 2.21 0 4 1.79 4 4s-1.79 4-4 4c-.43 0-.84-.09-1.23-.21-.03-.01-.06-.02-.1-.03A5.98 5.98 0 0 0 15 8zm1.66 5.13C18.03 14.06 19 15.32 19 17v3h4v-3c0-2.18-3.58-3.47-6.34-3.87zM9 6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2m0 9c-2.7 0-5.8 1.29-6 2.01V18h12v-1c-.2-.71-3.3-2-6-2M9 4c2.21 0 4 1.79 4 4s-1.79 4-4 4-4-1.79-4-4 1.79-4 4-4zm0 9c2.67 0 8 1.34 8 4v3H1v-3c0-2.66 5.33-4 8-4z']`;
-      const chatIconXpath = `//div[@aria-label='Chat with other participants']//*[@d='M20 2H4c-1.1 0-1.99.9-1.99 2L2 22l4-4h14c1.1 0 2-.9 2-2V4c0-1.1-.9-2-2-2zm0 14H4V4h16v12z']`;
+      const participantsIconXpath = `//div[@aria-label='Show everyone']//*[@d='M15 8c0-1.42-.5-2.73-1.33-3.76.42-.14.86-.24 1.33-.24 2.21 0 4 1.79 4 4s-1.79 4-4 4c-.43 0-.84-.09-1.23-.21-.03-.01-.06-.02-.1-.03A5.98 5.98 0 0 0 15 8zm1.66 5.13C18.03 14.06 19 15.32 19 17v3h4v-3c0-2.18-3.58-3.47-6.34-3.87zM9 6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2m0 9c-2.7 0-5.8 1.29-6 2.01V18h12v-1c-.2-.71-3.3-2-6-2M9 4c2.21 0 4 1.79 4 4s-1.79 4-4 4-4-1.79-4-4 1.79-4 4-4zm0 9c2.67 0 8 1.34 8 4v3H1v-3c0-2.66 5.33-4 8-4z']`;
+      const chatIconXpath = `//div[@aria-label='Chat with everyone']//*[@d='M20 2H4c-1.1 0-1.99.9-1.99 2L2 22l4-4h14c1.1 0 2-.9 2-2V4c0-1.1-.9-2-2-2zm0 14H4V4h16v12z']`;
 
       const participantsIcon = xpath(participantsIconXpath, document);
       const chatIcon = xpath(chatIconXpath, document);
```

**Comment by dzaman** [comment]
@g7nbp thanks for filing this issue! Version 3 of the extension should fix this and will be available in the Chrome Store after it is approved.

@Nu11u5 thanks for calling out the fix. I included your suggestion in commit 09ad7e9c69809a8c4e9a377df3a5c93d712e3ce7.
